authority-portal-frontend/src/app/common/components/form-elements/text-input/text-input.component.html

```diff
@@ -47,7 +47,7 @@
   <mat-error
     *ngIf="ctrl.touched && ctrl.hasError('phoneNumber')"
     class="text-xs">
-    Invalid phone number. Please use 5 or more numbers and optionally '+' or '
+    Invalid phone number. Please use 5 or more numbers and optionally '+' or
```

Comment:
This error message is not elegant imo
How about adding another validator to the field, e.g. the `minLength` validator. Then, you could split the error messages in "Invalid phone number. Only digits and '+' allowed" and "Phone number too short" respectively.